snappy.go

```diff
@@ -6,7 +6,7 @@
 // It aims for very high speeds and reasonable compression.
 //
 // The C++ snappy implementation is at https://github.com/google/snappy
-package snappy // import "github.com/golang/snappy"
```

Comment:
A good tutorial for this is https://splice.com/blog/contributing-open-source-git-repositories-go/